test/PR_test/integration_test/op/numpyop/meta/test_fuse.py

```diff
@@ -41,6 +47,14 @@ def instantiate_system():
 
         self.assertEqual(loaded_var, new_var)
 
+    def test_delete_op(self):
+        ops = fe.op.numpyop.meta.Fuse(
+            ops=[TestNumpyOp(inputs='x', outputs='x', mode="train", var=1), Delete(
```

Comment:
I think the real test we are interested in here is Fuse((a->a,b), (del b)) and make sure that it runs without requiring b as an input to Fuse

Reply:
Sure will add that test case too.

Reply:
generally it's better if unit tests are black-box such that if someone later came and messed up the code without realizing why it had been written that way the test would catch it